apps/meteor/app/cloud/server/functions/getWorkspaceAccessTokenWithScope.ts

```diff
@@ -8,10 +8,16 @@ import { CloudWorkspaceAccessTokenError } from './getWorkspaceAccessToken';
 import { removeWorkspaceRegistrationInfo } from './removeWorkspaceRegistrationInfo';
 import { retrieveRegistrationStatus } from './retrieveRegistrationStatus';
 
-export async function getWorkspaceAccessTokenWithScope(scope = '', throwOnError = false) {
+type WorkspaceAccessTokenWithScope = {
+	token: string;
+	expiresAt: Date;
+	scope: string;
+};
+
+export async function getWorkspaceAccessTokenWithScope(scope = '', throwOnError = false): Promise<WorkspaceAccessTokenWithScope> {
```

Comment:
Is it possible to avoid this boolean parameter? It'd be clearer to have an object with named props instead.

Reply:
It is, but I wanted to avoid touching that much code on a regression fix, wdyt?

Reply:
agreed